CodegenCLI/README.md

```diff
@@ -0,0 +1,60 @@
+# CodegenCLI
+
+```
+OVERVIEW: A command line utility for Apollo iOS code generation.
+
+USAGE: apollo-ios-codegen <subcommand>
+
+OPTIONS:
+  --version               Show the version.
+  -h, --help              Show help information.
+
+SUBCOMMANDS:
+  init                    Initialize a new configuration with defaults.
+  validate                Validate a configuration file or JSON formatted string.
+  generate                Generate Swift source code based on a code generation configuration.
+
+  See 'apollo-ios-codegen help <subcommand>' for detailed help.
+```
+
+## Initialize
+```
+OVERVIEW: Initialize a new configuration with defaults.
+
+USAGE: apollo-ios-codegen init [--output <output>] [--path <path>] [--overwrite]
+
+OPTIONS:
+  -o, --output <output>   Destination for the new configuration. (default: file)
```

Comment:
Do we need this? If they provide a path, we can assume they want to write it to a file at that path. If they don't assume we are printing it to stdout.

Maybe in that case, change the `--path` option to be the `--output` option and get rid of the existing output option?